app/models/checkbox_criterion.rb

```diff
@@ -177,6 +177,20 @@ def self.load_from_yml(criterion_yml)
     # a description is not given.
     criterion.description =
       criterion_yml[1]['description'].nil? ? '' : criterion_yml[1]['description']
+    # Visibility options
+    criterion.ta_visible = criterion_yml[1]['ta_visible'] unless criterion_yml[1]['ta_visible'].nil?
+    criterion.peer_visible = criterion_yml[1]['peer_visible'] unless criterion_yml[1]['peer_visible'].nil?
     criterion
   end
+
+  # Returns a hash containing the information of a single checkbox criterion.
+  def self.to_yml(criterion)
+    criterion_info = {}
```

Comment:
Make this a single expression, which is a hash literal:

```
{ "#{criterion.name}" => {
  type: 'checkbox',
  ...
}}
```

If symbol keys don't work, you can keep using strings.